demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -99,11 +100,25 @@ class ScriptType(StrEnum):
 class StructureError(BaseStrictModel):
     """Used for wrapping Pydantic errors, not part of content."""
 
+    path: Path
     field_name: Optional[tuple] = Field(None, alias="loc")
     error_message: Optional[str] = Field(None, alias="msg")
     error_type: Optional[str] = Field(None, alias="type")
     ctx: Optional[dict] = None
 
+    def __str__(self):
+        if self.error_type == "assertion_error":
+            error_message = f"The field {self.field_name} is not required, but should not be None if it exists"
```

Comment:
the tuple outcome (`The field ('name',) is not required, but should not be None if it exists`) can be prettier;

replace `self.field_name` with `",".join(more_itertools.always_iterable(self.field_name))`, this way if it's one (the common case) we'll get just the field name, otherwise we'll get comma-separated values. 

Since we access `self.field_name` multiple times (in the different branches of this condition), save the `",".join(...)` as a variable and use it instead.

Reply:
cool